lua/autorun/server/sv_spawn_protection.lua

```diff
@@ -187,11 +187,7 @@ end
 
 -- Prevents damage if a player has spawn protection
 local function preventDamageDuringSpawnProtection( player, damageInfo )
-
-    if ( playerHasSpawnProtection( player ) ) then
-        damageInfo:SetDamage( 0 )
-        return false
-    end
+    return true
```

Comment:
:( you forgot the conditional

`if playerHasSpawnProtection( player ) then return true end`